docs/tools/vdb_table/data/rockset.json

```diff
@@ -2,7 +2,7 @@
   "name": "Rockset",
   "links": {
     "docs": "https://docs.rockset.com/documentation/docs/vector-search",
-    "github": "https://github.com/rockset",
+    "github": "https://github.com/rockset/community/tree/master",
```

Comment:
This is supposed to be the organization's github handle if the DB is not open-source. You can keep it as ""https://github.com/rockset"